tiledb/ml/models/tensorflow_keras.py

```diff
@@ -385,3 +401,19 @@ def _load_weights_from_tiledb(
                 )
             var_value_tuples.extend(zip(weight_vars, read_weight_values))
         backend.batch_set_value(var_value_tuples)
+
+    def _get_tensorboard_files(self, log_dir: str) -> Mapping[str, bytes]:
+        """
+        This function accepts a Path log directory, evaluates path existense and returns a dictionary of TB log event files
```

Comment:
Don't really need a docstring for a self-explanatory private method (no other method in this class has one). Also please limit the length to ~90 chars per line (same as code).

Reply:
I thought our pre-commit hooks would take care of it, but as you said it is overkill so I ll remove. thanks